docs/source/installation/mac.rst

```diff
@@ -3,12 +3,22 @@ Mac
 
 First, you need to install Homebrew.  Homebrew is a package manager similar to
 ``apt`` or ``pacman`` on linux that allows you to install packages easily.  If
-it is not installed, please install it with this command:
+it is not installed, please install it with this command (for Apple Silicon useres, please read first the tip):
 
 .. code-block:: bash
 
    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" < /dev/null 2> /dev/null
 
+
+.. tip:: 
+
+    As of February 2020, the installation with ``brew`` had some issues with scipy and cairo packages in Apple Silicon mode. This can be solved by installing brew in the rosetta2 mode.
```

Comment:
```suggestion
    As of February 2020, the installation with ``brew`` has some issues with scipy and cairo packages in Apple Silicon mode. This can be solved by installing brew in the rosetta2 mode.
```